build/optaplanner-build-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
     <version.org.apache.logging.log4j>2.20.0</version.org.apache.logging.log4j>
     <version.com.thoughtworks.xstream>1.4.20</version.com.thoughtworks.xstream>
     <version.io.quarkiverse.operatorsdk>5.1.4</version.io.quarkiverse.operatorsdk>
-    <version.io.quarkus>2.13.8.Final</version.io.quarkus>
+    <version.io.quarkus>2.16.10.Final</version.io.quarkus>
```

Comment:
productized Quarkus is 2.13.9.Final so this upgrade in this branch doesn't make sense